src/plugin/plugin.ts

```diff
@@ -620,11 +652,25 @@ class Purchases {
       null,
       PLUGIN_NAME,
       "configure",
-      [apiKey, appUserID, observerMode, userDefaultsSuiteName]
+      [apiKey, appUserID, observerMode, userDefaultsSuiteName, useAmazon]
     );
     this.setupShouldPurchasePromoProductCallback();
   }
 
+  /**
+   * Sets up Purchases with your API key and an app user id.
+   * @param {PurchasesConfiguration} Object containing configuration parameters
+   */
+   public static configureWith({
+    apiKey,
+    appUserID = null,
+    observerMode = false,
+    userDefaultsSuiteName,
+    useAmazon = false
+  }: PurchasesConfiguration): void {
```

Comment:
We probably need to add this and the `PurchaseConfiguration` class to the api tests.